### PR TITLE
Use supplied certbot timer

### PR DESCRIPTION
**Pull request by denibertovic**
and make sure to reload nginx on renewals

### CODE REVIEW

**Comment by denibertovic** [comment]
ping @mrgnr

File: letsencrypt/tasks/main.yml [hunk 1]
```diff
@@ -57,11 +57,17 @@
...
-  cron: >
-    name="Let's Encrypt certificate renewal"
-    cron_file="letsencrypt"
-    special_time="daily"
-    user="root"
-    job="{{ letsencrypt_command }} renew --agree-tos"
-  tags: ['letsencrypt']
+- name: Make sure certbot systemd override dir exists
+  file: path=/etc/systemd/system/certbot.service.d state=directory
+  tags:
```
Comment (mrgnr) [inline letsencrypt/tasks/main.yml]: Do you need a `>` here?

Comment (denibertovic) [inline letsencrypt/tasks/main.yml]: Nope. Should work just fine like this.

**Comment by denibertovic** [comment]
@mrgnr I have this running and installed...I just copied over from my own roles  so I'm pretty confident this exists and actually works.

**Comment by denibertovic** [comment]
@mrgnr aaaaand I'm not running 16.04.... 16.04 doesn't actually have the certbot package.... lovely.

**Comment by denibertovic** [comment]
@mrgnr updated with the correct package that has the latest and greatest. Had to add a ppa as per the official letencrypt instructions.

**Comment by mrgnr** [comment]
@denibertovic Could you point me to the docs on this? I installed the package from the ppa in a docker container, and I don't see a timer. I do see a cron job in `/etc/cron.d/certbot` though:

```
0 */12 * * * root test -x /usr/bin/certbot -a \! -d /run/systemd/system && perl -e 'sleep int(rand(3600))' && certbot -q renew
```

**Pull request merged**